pages/about.html

```diff
@@ -4,9 +4,40 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="/styles/style.css" rel="stylesheet">
+ 
 </head>
 <body>
-  
-</body>
+<section class = "flex-container">
```

Comment:
In HTML, prefer no spaces around the attribute `=`.

Reply:
Because of the flex styling on this element, the Who is She? text ends up getting pushed off the screen. You were able to use the flex layout to get some centering, but make sure that it's not having any other side-effects.

Reply:
`section` should not generally be used for the main content of the page. From MDN:
>`<section>` is a generic sectioning element, and should only be used if there isn't a more specific element to represent it. As an example, a navigation menu should be wrapped in a [`<nav>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) element, but a list of search results or a map display and its controls don't have specific elements, and could be put inside a `<section>`.